followthemoney/schema/CallForTenders.yaml

```diff
@@ -0,0 +1,77 @@
+CallForTenders:
+  label: "Call for tenders"
+  plural: "Calls for tenders"
+  description: "A public appeal issued by an authority, possibly on behalf of another, for buying a specific work, supply or service"
+  extends:
+    - Thing
+  required:
+    - title
+  featured:
+    - title
+    - contractingAuthority
+  caption:
+    - title
+  properties:
+    title:
+      label: "Title"
+    uid:
+      label: "CfT Unique ID"
+    cftCaUid:
+      label: "CfT CA Unique ID"
```

Comment:
That seems super super specific. Really needed? 
Btw all ID fields should be `type: identifier`

Reply:
I already asked the same question and it seems so. I'll change the type though yes, thanks.